src/image_bios.cpp

```diff
@@ -30,6 +30,14 @@ static constexpr auto spiDriver = "1e631000.spi";
 static const char* mtdDevice = "/dev/mtd/bios";
 static constexpr auto gpioNamePCHPower = "PWRGD_DSW_PWROK";
 static constexpr auto gpioNameBIOSSel = "MIO_BIOS_SEL";
+
+#ifdef GOLDEN_FLASH_SUPPORT
+extern bool useGoldenFlash; // Defined in main.cpp
+static constexpr auto gpioNameActiveFlashSel = "SPI_BIOS_ACTIVE_FLASH_SEL";
+static constexpr auto mainFlashDrive = 0;
+static constexpr auto goldenFlashDrive = 1;
```

Comment:
I think both must be *Chip instead of *Drive

Reply:
fixed